tests/tester.py

```diff
@@ -136,11 +138,16 @@ def filter_marks(marks, nested):
                 testinputpy = fp.read()
 
             pymarks = []
-            mobj = re.search(r'^@ctest_labeler\("' + r"(?P<pymarks>([a-z0-9-;]+))" + r'\"\)', testinputpy, re.MULTILINE)
+            pyplugins = []
+            mobj = re.search(r'^@uusing\("' + r"(?P<pyplugins>([a-z0-9-_;]+))" + r'\"\)', testinputpy, re.MULTILINE)
+            if mobj:
+                pyplugins = mobj.group("pyplugins").split(";")
```

Comment:
Thanks for catching, will fix. (I think the only case where this bit is currently does anything is the ecpint tests since other plugin tests are nested in folders, which is already dealt with)